instrumentation/apolloconfig/apolloconfig-2.0.0/testing/src/main/java/io/opentelemetry/instrumentation/apolloconfig/v2_0_0/ApolloRepositoryChangeTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.apolloconfig.v2_0_0;
+
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.equalTo;
+
+import com.ctrip.framework.apollo.enums.ConfigSourceType;
+import com.ctrip.framework.apollo.internals.AbstractConfigRepository;
+import com.ctrip.framework.apollo.internals.ConfigRepository;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.sdk.testing.assertj.AttributeAssertion;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public abstract class ApolloRepositoryChangeTest {
```

Comment:
Since this test class is abstract it is probably not run at. `testing` module is usually used when you need to share testing code between multiple modules or when putting the tests in the same module with the implementation does not work for some reason. Here you should move the testing code to the javaagent module, this will also ensure that your tests actually run with the instrumentation that you wrote.